client/lib/user/user.d.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * External Dependencies
+ */
+import { EventEmitter } from 'events';
+
+export interface User extends EventEmitter {
+	initialize: () => Promise< void >;
+	clearStoreIfChanged: () => void;
+	get: () => UserData;
+	fetch: () => Promise< void > | boolean;
+	handleFetchFailure: ( error: Error ) => void;
+	handleFetchSuccess: ( userdata: UserData ) => void;
+	getLanguage: () => any | undefined;
+	getAvatarUrl: ( options: any ) => string;
+	clear: () => Promise< void >;
```

Comment:
if Redux persistence isn't enabled this returns `undefined`

```ts
clear: () => Promise<void> | void;
```

Reply:
ah good catch